tests/mir_test_doubles/mock_libinput.cpp

```diff
@@ -823,6 +823,10 @@ libinput_event* mtd::MockLibInput::setup_axis_event(libinput_device* dev, uint64
         .WillByDefault(Return(vertical));
     ON_CALL(*this, libinput_event_pointer_get_axis_value_discrete(pointer_event, LIBINPUT_POINTER_AXIS_SCROLL_HORIZONTAL))
         .WillByDefault(Return(horizontal));
+    ON_CALL(*this, libinput_event_pointer_get_axis_value(pointer_event, LIBINPUT_POINTER_AXIS_SCROLL_VERTICAL))
+        .WillByDefault(Return(vertical));
+    ON_CALL(*this, libinput_event_pointer_get_axis_value(pointer_event, LIBINPUT_POINTER_AXIS_SCROLL_HORIZONTAL))
+        .WillByDefault(Return(horizontal));
```

Comment:
Note that even on discrete events, the non-discrete functions still return values. This matches the libinput implementation